### PR TITLE
Add a detailed guide on how to how contributing to this project in the Readme.md file

### PR DESCRIPTION
**Pull request by Sohoxic**
Hey I've worked on issue #4 . I've added detailed information regarding contribution and added necessary hyperlinks related to the same. I've also generated the list of contributors and added it in the end. I've created a folder called assets/images for all the images used in the readme.md file.
Thank you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,8 +13,106 @@ A community of new developers buzzing around and collaborating on beginner-frien
...
-[Contribution guidelines, code of conduct, etc.]
-
+  ### 🔖Steps to Contribute
+
+Following are the steps to guide you:
+
+* Step 1: Fork the repo Go to your Git terminal and clone it on your machine.
+* Step 2: Add an upstream link to the main branch in your cloned repo
+    ```
+    git remote add upstream https://github.com/SarthakSKumar/IoT-Projects-and-Scripts.git
```
Comment (ArslanYM) [inline README.md:23]: Remove your name and add  [Your Username]

**Review by ArslanYM — CHANGES_REQUESTED** [review]
Change the mentioned details.

**Comment by ArslanYM** [comment]
@Sohoxic Love your enthusiasm for this project but try to solve only the given issue. I see that you have made changes which are irrelevant the the actual issue. Thankyou anyways. Consider this from next time

**Comment by Sohoxic** [comment]
I have made the necessary changes. Thank You

**Review by ArslanYM — APPROVED** [review]

**Comment by ArslanYM** [comment]
@Sohoxic LGTM 👍

**Pull request merged**